js-miniapp-sdk/README.md

```diff
@@ -162,6 +162,34 @@ miniApp.shareInfo(info)
     .catch(error => console.error(error));
 ```
 
+### 5. Requesting User details
+
+You can retrieve the User Name and Profile Photo of the user using the following interfaces. Please make sure that User have allowed respective custom permission before requesting the user detail.
+
+```javascript
+const info = { content: inputValue };
+
+MiniApp.getUserName()
+    .then(userName => {
+		console.log(userName);
+	}).catch(error => {
+		console.error(error);
+	});
+```
+
+NOTE: getProfilePhoto() -  Returns the Base64 Encoded image string from the Host app.
```

Comment:
Is it possible to include any convention or example as a guideline for the host app e.g.

```
data:image/png;base64,XXX
```

Reply:
oops now I noticed this. It's possible for the hostapp to return the net url for the profile photo so I think it can be 
```suggestion
NOTE: getProfilePhoto() -  Returns the image uri from the Host app.
```
The miniapp can load both http url or base64 string (with correct format).

@ts-leojoseph-r @khairul-alam-licon in our android & ios sample app, we return the base64 string so we need to pass it with the form `data:image/png;base64, XXX`.

Reply:
That's right @minh-rakuten . We are alreaddy returning as base 64 encoded string from iOS to the miniapp. Thanks for the suggestion, updated the comments.